src/core/os_android.c

```diff
@@ -280,27 +280,27 @@ void os_poll_events() {
   }
 }
 
-void os_on_quit(quitCallback callback) {
+void os_on_quit(fn_quit callback) {
```

Comment:
Also I think the function pointers here should have a star after them e.g. `fn_quit*`

Reply:
You are right, as I wrote them these functions would crash if called